MapboxCoreNavigation/DistanceFormatter.swift

```diff
@@ -131,6 +131,16 @@ open class DistanceFormatter: LengthFormatter {
     
     /// Indicates the most recently used unit
     public private(set) var unit: LengthFormatter.Unit = .millimeter
+	
+    public var locale: Locale {
+        set {
+            self.numberFormatter.locale = newValue
+            self.nonFractionalLengthFormatter.numberFormatter.locale = newValue
```

Comment:
You mean teh numberFormatter and the nonFractionalLengthFormatter were not synced before?